### ISSUE TITLE
Too easy to accidentally get converted to student account

### ISSUE THREAD

**Issue by smloh** [codecombat/codecombat#6250]
I got converted to a student account just by clicking on a link someone sent me even though I didn't intend to.
I wasn't able to find a way to undo it except by contacting customer support.

This could have been avoided with a confirmation dialog before switching an existing account to a student account, or by having a way to let people switch back in their profile page.

**Comment by AndrewJakubowicz** [comment]
Hi smloh,

Thanks for opening an issue. Do you have a copy of the link that did this conversion into a student account.

Thanks,
Andrew

**Comment by smloh** [comment]
Yes, the link text was "Code Combat - [name of team]" and the link url was https://codecombat.com/students?_cc=BurnLionMouth

**Comment by AndrewJakubowicz** [comment]
That link is very helpful thank you (you can keep it obfuscated)!
And to clarify, you didn't want to join the class. You clicked the url and it automagically made you a student.

I'll take a look thank you!

**Comment by smloh** [comment]
> And to clarify, you didn't want to join the class. You clicked the url and it automagically made you a student.

Yes that is correct, with no indication that was what was going to happen.
